### ISSUE TITLE
Include raw test file contents

### ISSUE THREAD

**Issue by rwaldron** [bocoup/test262-stream#2]
In addition to the `contents` property, is it possible to also include the "uncompiled" source of the test file?

**Comment by adrianheine** [comment]
Yes, please! I currently resort to [stripping out](https://github.com/adrianheine/test262-parser-runner/blob/master/lib.js) the helpers afterwards to (massively) reduce parse time.

**Comment by adrianheine** [comment]
This should be doable now with #7 landed, right? I think for me it would be even better to have an option to not add the helpers in the first place.
